ob-pattern-5/templates/obiam/wso2ob-pattern-5-obiam-service.yaml

```diff
@@ -0,0 +1,27 @@
+# ----------------------------------------------------------------------------------------
+#
+# Copyright (c) 2023, WSO2 LLC. (https://www.wso2.com). All Rights Reserved.
+#
+# This software is the property of WSO2 LLC. and its suppliers, if any.
+# Dissemination of any information or reproduction of any material contained
+# herein in any form is strictly forbidden, unless permitted by WSO2 expressly.
+# You may not alter or remove any copyright or other notice from copies of this content.
+#
+# ----------------------------------------------------------------------------------------
+
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ template "ob-pattern-5.resource.prefix" . }}-obiam-service
+  namespace : {{ .Release.Namespace }}
+spec:
+  # label keys and values that must match in order to receive traffic for this service
+  selector:
+    deployment: {{ template "ob-pattern-5.resource.prefix" . }}-obiam
+  ports:
+  - name: servlet-https
+    port: 9446
+    protocol: TCP
+
```

Comment:
@Chathuratharushka please remove these unnecessary, extra EOF lines - only one is enough.

Reply:
done https://github.com/wso2/kubernetes-open-banking/pull/76/commits/2d17757e2bd4810ba794310a484bc83ef45d718a